OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -19,6 +19,7 @@
 
 namespace OpenRA.Mods.Common.Warheads
 {
+	[Desc("Spawn a temporary image.")]
```

Comment:
The description feels a bit weird. Also, fwiw, it allows to play sounds as well.

Reply:
I'm going with "Spawn a sprite with sound."